tests/problems/generic/test_fgw_problem.py

```diff
@@ -252,3 +253,51 @@ def test_prepare_different_costs(self, adata_time: AnnData):
         assert isinstance(problem[0, 1].xy.cost, Cosine)
         assert isinstance(problem[0, 1].x.cost, Euclidean)
         assert isinstance(problem[0, 1].y.cost, SqEuclidean)
+
+    @pytest.mark.parametrize(("memory", "refresh"), [(1, 1), (5, 3), (7, 5)])
+    @pytest.mark.parametrize("recenter", [True, False])
+    def test_passing_ott_kwargs_linear(self, adata_space_rotate: AnnData, memory: int, refresh: int, recenter: bool):
+        problem = GWProblem(adata=adata_space_rotate)
+        problem = problem.prepare(
+            key="batch",
+            policy="sequential",
+            joint_attr="X_pca",
+            x_attr={"attr": "obsm", "key": "spatial"},
+            y_attr={"attr": "obsm", "key": "spatial"},
+        )
+
+        problem = problem.solve(
```

Comment:
How long does this take locally? 
As we can test this without convergence of the solver, we can set `max_iterations=1`.

Reply:
with max_iterations=1 it takes 1 min

Reply:
in Sinkhorn it takes 20 sec

Reply:
ok can you also set the number of inner iterations to a smaller value please? E.g. also 1

Reply:
done, down to 40 sec